project/CrossJava.scala

```diff
@@ -56,14 +56,16 @@ object CrossJava {
     val discoveredJavaHomes = settingKey[Map[String, File]]("Discovered Java home directories")
     val javaHomes = settingKey[Map[String, File]]("The user-defined additional Java home directories")
     val fullJavaHomes = settingKey[Map[String, File]]("Combines discoveredJavaHomes and custom javaHomes.")
+    val useSystemJdk = settingKey[Boolean]("Whether to use system jdk instead of explicit jdk8")
```

Comment:
Perhaps make it clear what for: `useSystemJdkBootClasspath`? And perhaps here again make it clear this means the resulting artifacts will not work on jdk8?